src/server_manager/web_app/ui_components/outline-per-key-data-limit-dialog.ts

```diff
@@ -197,6 +193,8 @@ export class OutlinePerKeyDataLimitDialog extends LitElement {
   private renderMenu() {
```

Comment:
Perhaps `renderDataLimit` to align the language?

Reply:
Done in 3a84a661f51d59193d732ae58743c5e6c7075746